cfme/tests/infrastructure/test_vm_engine_relationship.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from cfme import test_requirements
+from cfme.infrastructure.provider import InfraProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.generators import random_vm_name
+from cfme.utils.wait import wait_for
+
+pytestmark = [
+    pytest.mark.tier(2),
+    pytest.mark.provider(classes=[InfraProvider], selector=ONE_PER_TYPE),
+    pytest.mark.usefixtures('setup_provider'),
+    test_requirements.rhev,
```

Comment:
Do we want to mark cases with vmware/scvmm/ providers with the rhev requirement marker, since we're collecting against InfraProvider generically?

Reply:
not really. I marked it with general UI too as this is what is says in the BZ